sky/shell/platform/ios/accessibility_bridge.h

```diff
@@ -0,0 +1,177 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+#define SKY_SHELL_PLATFORM_IOS_SEMANTICS_LISTENER_IMPL_H_
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/macros.h"
+#include "base/memory/ref_counted.h"
+#include "base/memory/weak_ptr.h"
+#include "mojo/public/cpp/bindings/array.h"
+#include "mojo/public/cpp/bindings/strong_binding.h"
+#include "sky/engine/platform/geometry/FloatRect.h"
+#include "sky/services/semantics/semantics.mojom.h"
+#include "sky/shell/platform/ios/FlutterView.h"
+#include "third_party/skia/include/core/SkRect.h"
+#include "third_party/skia/include/utils/SkMatrix44.h"
+
+namespace sky {
+namespace shell {
+namespace a11y {
+
+class Node;
```

Comment:
I'd probably make this an inner class for AccessibilityBridge given that it looks like an internal concept for that class.